system/Test/FeatureTestCase.php

```diff
@@ -298,7 +298,7 @@ protected function setupRequest(string $method, string $path = null, array $para
 		$config = config(App::class);
 		$uri    = new URI($config->baseURL . '/' . trim($path, '/ '));
 
-		$request      = new IncomingRequest($config, clone($uri), $params, new UserAgent());
+		$request      = new IncomingRequest($config, clone($uri), $params ?? 'php://input', new UserAgent());
```

Comment:
Since the FeatureTestCase class is always calling IncomingRequest from within code, is there any point to using `php://input` here?

Reply:
Needed when sending a request body to json